api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -74,7 +77,26 @@ public void validate(Object target, Errors errors) {
 		        && OpenmrsUtil.compareWithNullAsLatest(visit.getStartDatetime(), visit.getStopDatetime()) > 0) {
 			errors.rejectValue("stopDatetime", "Visit.error.endDateBeforeStartDate");
 		}
-		
+		try 
+		{
+			Date birthDate = visit.getPatient().getBirthdate(); 
+			Date startDateTime = visit.getStartDatetime();    
+			String dateFormat = "dd-MM-YYYY"; 
```

Comment:
The formatting here does not look correct. Did you take a look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips